fonduer/parser/parser.py

```diff
@@ -504,10 +512,10 @@ def _parse_paragraph(self, node, state):
             state["paragraph"]["text"] = text
             state["paragraph"]["field"] = field
 
-            # Parse the Sentences in the Paragraph
-            yield from self._parse_sentence(paragraph, node, state)
-
-        return state
+            all_sentences = [y for y in self._parse_sentence(paragraph, node, state)]
+            for sen in all_sentences:
+                yield sen
```

Comment:
Do we need this `all_sentences`? Seems more efficient to yield directly.

Reply:
No, we can yield instead. This is a remnant of me timing the complete sentence generation.